cli/azd/pkg/tools/python/python.go

```diff
@@ -119,13 +128,25 @@ func (cli *PythonCli) CreateVirtualEnv(ctx context.Context, workingDir, name str
 	return nil
 }
 
-func pythonExe() string {
+func checkPath() (string, bool, error) {
+	var found bool
+	var err error
 	if runtime.GOOS == "windows" {
-		if _, err := osexec.LookPath("py"); err != nil {
-			return "python"
+		// py for https://peps.python.org/pep-0397
+		pyString := [2]string{"py", "python"}
```

Comment:
Should comment here: `// order is important. we want to resolve 'py', if available, first`